src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import './NewTodo.scss';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types';
+import { titlePattern } from '../../RegExps';
+import { getUserById } from '../../utils';
+import { getNewTodoId } from '../../utils';
+
+type Props = {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
```

Comment:
i think it will be better)
```suggestion
  setTodos: (todos: Todo[]) => void;
```

Reply:
Kyrylo told us today, that `(todos: Todo[]) => void;` is an incorrect option, because almost every function fits this option👀